agent/src/beerocks/slave/agent_ucc_listener.cpp

```diff
@@ -231,3 +233,23 @@ bool agent_ucc_listener::handle_dev_set_config(std::unordered_map<std::string, s
  * been received.
  */
 std::string agent_ucc_listener::get_selected_backhaul() { return m_selected_backhaul; }
+
+bool agent_ucc_listener::get_mac_by_ssid(const std::string &ruid, const std::string &ssid,
```

Comment:
This function should be moved to the DB.
It has nothing to do as part of the `agent_ucc_listener`.
Evidence: it uses no member of the class, nor it has an "application logic".

Reply:
Good point, it could be moved to the db

Reply:
we already have db::get_hostap_vap_with_ssid